### PR TITLE
feat: adds some friendly tips

### LINKED ISSUE #249: feat: tips
adds some friendly tips describing things about hilbish like features, documentation, etc. this can also be done on first time run to link to the getting started page on the website and config location.

this should be done similarly to how the motd and greeting are shown on startup. a simple table of tips randomly printed would work

### PR DESCRIPTION
**Pull request by nhlmg93**
describing things about hilbish like fe…atures, documentation, etc.

---
- [ X] I have reviewed CONTRIBUTING.md.
- [X ] My commits and title use the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format.
- [ X] I have documented changes and additions in the CHANGELOG.md.
---

Closes #249

### CODE REVIEW

**Comment by nhlmg93** [comment]
Let me know if i need to change, remove, or add anything.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## [2.3.0] - 2024-07-20
 ### Added
+- `nature.opts.tips` was added to randomly generate tips on start up.
+Follows the pattern of motd and greeting. Displays after greeting on start up.
+### Added
```
Comment (TorchedSammy) [inline CHANGELOG.md:7]: This should be added under an "Unreleased" heading:

```suggestion
## Unreleased
### Added
- `nature.opts.tips` was added to randomly generate tips on start up.
Follows the pattern of motd and greeting. Displays after greeting on start up.

## [2.3.0] - 2024-07-20
### Added
```

File: nature/opts/tips.lua [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Getting Started: https://rosettea.github.io/Hilbish/docs/getting-started/
+Documentation: https://rosettea.github.io/Hilbish/
+Github: https://github.com/Rosettea/Hilbish
+]]
+
+hilbish.tips = {
+	"Join the discord and say hi! -> https://discord.gg/3PDdcQz",
+	"{green}hilbish.alias{reset} -> Sets an alias to another cmd",
+	"{green}hilbish.appendPath{reset} -> Appends the provided dir to the command path ($PATH)",
+	"{green}hilbish.appendPath{reset} -> Appends the provided dir to the command path ($PATH)",
```
Comment (TorchedSammy) [inline nature/opts/tips.lua:14]: duplicate

File: nature/opts/tips.lua [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	[[
+	Add Lua-written commands with the commander module! 
+	Checkout the docs here -> https://rosettea.github.io/Hilbish/docs/api/commander/
+	]]
+}
+
+bait.catch('hilbish.init', function()
+	if hilbish.interactive and hilbish.opts.tip then
+		local idx = math.random(1, #hilbish.tips)
+		print(lunacolors.format(PREAMBLE .. "\nTip: " .. hilbish.tips[idx]))
```
Comment (TorchedSammy) [inline nature/opts/tips.lua:24]: Honestly.. I would either shorten or remove the preamble. A single link to the website serves to link the GitHub and the documentation (including the getting started doc).

**Review by TorchedSammy — CHANGES_REQUESTED** [review]
Thanks for the PR. A couple changes for you to make. Might I also suggest adding a few more tips, like ones related to everything under the "Features" heading on the docs? ie runner mode

**Review by nhlmg93 — COMMENTED** [review]
move change to unreleased in the change log
added more hints especially around hilbish features
shorted preamble to just Getting Started with a link to the respective page in the docs.

File: nature/opts/tips.lua [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	[[
+	Add Lua-written commands with the commander module!
+	Checkout the docs here -> https://rosettea.github.io/Hilbish/docs/api/commander/
+	]]
+}
+
+bait.catch('hilbish.init', function()
+	if hilbish.interactive and hilbish.opts.tip then
+		local idx = math.random(1, #hilbish.tips)
+		print(lunacolors.format(PREAMBLE .. "\nTip: " .. hilbish.tips[idx]))
```
Comment (TorchedSammy) [inline nature/opts/tips.lua:29]: I just thought of this, but can you add a message that says "🛈 These tips can be disabled with `hilbish.opts.tips = false`" and color it yellow maybe

Comment (nhlmg93) [reply nature/opts/tips.lua:29]: would you like that to show up ever time in the preamble or would you want it to be a hint that shows some of the time?

Comment (TorchedSammy) [reply nature/opts/tips.lua:29]: it should show up all the time.

**Review by TorchedSammy — APPROVED** [review]

**Comment by TorchedSammy** [comment]
thanks for the contribution!

**Pull request merged**